turbopack/crates/turbo-tasks-backend/src/backend/storage.rs

```diff
@@ -337,7 +337,7 @@ where
 {
     pub fn new() -> Self {
         let shard_amount =
-            (available_parallelism().map_or(4, |v| v.get()) * 64).next_power_of_two();
+            (available_parallelism().map_or(4, |v| v.get()) * 256).next_power_of_two();
```

Comment:
This is a very aggressive number of shards. Is this change intended to be part of this PR?

Reply:
removed it